history/universe/marathon.json.mustache

```diff
@@ -57,7 +57,7 @@
     "healthChecks": [
         {
             "protocol": "COMMAND",
-            "command": { "value": "if [ -n \"$(tail -n500 $MESOS_SANDBOX/stderr | grep ERROR )\" ]; then echo 'Spark History Server logs contain errors'; exit 1; else exit 0; fi" },
+            "command": { "value": "curl --fail -s -o /dev/null  \"http://$LIBPROCESS_IP:$PORT0/api/v1/version\"" },
```

Comment:
I'd drop the inside quotes, I don't think we need them, since neither env var is allowed to contain spaces.
Also, it might be good to use the long options rather than short for clarity.

Reply:
done